content/kubeone/master/guides/local/_index.en.md

```diff
@@ -0,0 +1,62 @@
++++
+title = "Local subcommand"
+date = 2022-08-05T15:58:00+02:00
+enableToc = true
++++
+
+## Introduction
+
+`kubeone local` subcommands helps you to initialize a **local** (e.g. on the
+local machine, in the current shell), 1 node all-in-one cluster.
+
+## Use Cases
+
+Such a setup could be useful for cases such as:
+* Single edge server
+* Developer VM (kinda competing with kind but without placing everything in
```

Comment:
I would add a warning about this use case that this is possible, but that you have to be very careful because KubeOne modifies various system settings.

Reply:
done